patroni/postgresql/config.py

```diff
@@ -1169,7 +1170,7 @@ def reload_config(self, config: Dict[str, Any], sighup: bool = False) -> None:
                     pending_restart = self._postgresql.query(
                         'SELECT COUNT(*) FROM pg_catalog.pg_settings'
                         ' WHERE pg_catalog.lower(name) != ALL(%s) AND pending_restart',
-                        [n.lower() for n in self._RECOVERY_PARAMETERS])[0][0] > 0
+                        [n for n in params_skip_changes])[0][0] > 0
```

Comment:
Here we actually do need lower case comparison.
```suggestion
                        [n.lower() for n in params_skip_changes])[0][0] > 0
```

Reply:
🤦‍♀️  went too far